### PR TITLE
feat/storing votes as points and adding percision

### PR DESCRIPTION
**Pull request by RonTuretzky**
#29  & #31

- [ ] After #27 is merged, add precision handling for the view methods as mentioned [here](https://github.com/BreadchainCoop/breadchain/pull/27#discussion_r1582244666)

### CODE REVIEW

**Comment by RonTuretzky** [comment]
@bagelface , voting & distribution should be permissionless with no access control, that was the original design at least

File: src/YieldDisburser.sol [hunk 1]
```diff
@@ -15,15 +15,18 @@ contract YieldDisburser is OwnableUpgradeable {
     uint48 public lastClaimedTimestamp;
     uint256 public lastClaimedBlocknumber;
     uint48 public minimumTimeBetweenClaims;
+    uint256 public pointsMax;
     mapping(address => uint256[]) public holderToDistribution;
+    mapping(address => uint256) public holderToDistributionTotal;
+    uint256 public constant PERCISION = 1e18;
```
Comment (bagelface) [inline src/YieldDisburser.sol]: should be `PRECISION`

Comment (bagelface) [reply src/YieldDisburser.sol]: also feel like it can just be a var in local scope of the calculation

**Comment by bagelface** [comment]
Going to wait for https://github.com/BreadchainCoop/breadchain/pull/27 to be merged before approving this because of the precision issues pointed out in that PR

**Comment by RonTuretzky** [comment]
> Going to wait for #27 to be merged before approving this because of the precision issues pointed out in that PR

27 was merged and then this one was merged after

**Pull request merged**